### PR TITLE
{bio}[GCC/11.2.0] BEDTools v2.30.0

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Requires:
- [ ] [BAMTools](https://github.com/easybuilders/easybuild-easyconfigs/pull/14344)

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 18 out of 19 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/9d8876616ed94c2894308299d3c61fe6 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/f4d2a41e48151b3fc02ce8479a635408 for a full test report.

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.30.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+buildopts = 'CXX="$CXX"'
+
+files_to_copy = ["bin", "docs", "data", "genomes", "scripts", "test"]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bedtools', 'pairToBed', 'mergeBed', 'bedToBam', 'fastaFromBed']],
+    'dirs': files_to_copy,
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.30.0-GCC-11.2.0.eb:42]: @sassy-crick can we add a simple `sanity_check_command`, e.g. `--help` or `--version`?
```suggestion

sanity_check_commands = ['bedtools --help']

```

Comment (sassy-crick) [reply easybuild/easyconfigs/b/BEDTools/BEDTools-2.30.0-GCC-11.2.0.eb:44]: Good idea! I should do that from now on. That raises the question if we also should add that to the already merged PR [14344](https://github.com/easybuilders/easybuild-easyconfigs/pull/14344) ?

Comment (bedroge) [reply easybuild/easyconfigs/b/BEDTools/BEDTools-2.30.0-GCC-11.2.0.eb:44]: Ah, forgot to ask that for that one, indeed. Would have been useful, but we could do that in a separate/new PR, or for the next version of BamTools.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/610c89109f0a858d26317f09a8b4c597 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/aae29aa0700e25bfaff2ce52b7042877 for a full test report.

**Comment by bedroge** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**